content/guides/component-testing/component-framework-configuration.md

```diff
@@ -69,7 +69,7 @@ The Launchpad's setup wizard will do the following things:
 <!-- TODO: pic of all the files we make, collapsed -->
 
 Once you click through all of the prompts, you'll be asked to choose a browser
-and launch the Cypress App.
+and launch the Cypress Test Runner.
```

Comment:
Does the Test Runner itself open? Should we simplify to "...you'll be asked to choose a browser to continue."